src/NuGet.Clients/NuGet.SolutionRestoreManager/IVulnerabilitiesFoundService.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace NuGet.SolutionRestoreManager
+{
+    public interface IVulnerabilitiesFoundService
```

Comment:
I remember leaving comments or discussing in a PR review meeting that this interface name should be renamed to something like `IVulnerabilitiesNotificationService`.